src/main/java/io/cdap/plugin/gcp/bigquery/sqlengine/builder/BigQueryDeduplicateSQLBuilder.java

```diff
@@ -145,7 +145,7 @@ protected String getOrderByField(DeduplicateAggregationDefinition.FilterExpressi
     }
 
     // some_field ASC/DESC
-    return ((SQLExpression) filterExpression.getExpression()).extract() + SPACE + order;
+    return ((SQLExpression) filterExpression.getExpression()).extract() + SPACE + order + SPACE + NULLS_LAST;
```

Comment:
Please take a look at the unit tests in the `src/test/java/io/cdap/plugin/gcp/bigquery/relational/` directory and ensure that they are working after this change.